Server/ReasnAPI/ReasnAPI/Services/EventService.cs

```diff
@@ -203,17 +204,15 @@ public Event GetEventBySlug(string slug)
         return eventToReturn;
     }
 
-    public IEnumerable<ParticipantDto> GetEventParticipantsBySlug(string slug)
+    public int GetEventParticipantsBySlug(string slug, ParticipantStatus status)

```

Comment:
Name is inadequate, we are not geting only by slug and we are not getting participants

Reply:
Do we have then some possibility to get all participants for event if you have changed that to return only count?

Reply:
changed